### ISSUE TITLE
[RPP Back Office] Clean up Day Pass field

### ISSUE THREAD

**Issue by moorerst** [cityofaustin/atd-data-tech#4754]
Clean up task - update the Number of Permits and Passes  field with the correct value on existing applications where Day Passes were issued

**Comment by ChrispinP** [comment]
@moorerst Not sure I understand this task. The 'Number of Permits and Passes' field does not exist on the application. Its a redundant field that exists on the permit object when a permit is issued. It should only ever be 1 if used for reporting, if not used for reporting, the field should be removed from the object.

**Comment by moorerst** [comment]
@johnclary  when we discussed this issue following the meeting with the RPP team I may have gotten the field name wrong for this issue and issue # 4450 related to Day Passes.
Can you verify which field should be used for these issues?

**Comment by johnclary** [comment]
It's the "Total Day Passes" field that needs to be presented to the RPP staff and back-filled for previously issued permits.

**Comment by ChrispinP** [comment]
The Number of Permits and Passes field has been updated to Number of Permits, added to the Add Permit form and displays only when Day Passes are selected. I have updated the database to show correct values in this Number of Permits field for all permit types including some records that have no permit type. Since the Number of Day Passes field is now redundant and no longer needed, it has been removed.

Ready for review.
